usbsdmux/usbsdmux.py

```diff
@@ -147,6 +146,7 @@ class UsbSdMuxClassic(UsbSdMux):
 
     def __init__(self, sg):
         self._pca = Pca9536(sg)
+        self._usb = self._pca.get_usb()
 
```

Comment:
See the other comment on changing `Usb2642` / `Pca9536` initialization for more information.

```suggestion
    def __init__(self, sg):
        self._usb = Usb2642(sg)
        self._pca = Pca9536(self._usb)
```